language-support/ts/codegen/tests/BUILD.bazel

```diff
@@ -62,4 +62,7 @@ sh_test(
     deps = [
         "@bazel_tools//tools/bash/runfiles",
     ],
+    tags = [
+      "exclusive",
```

Comment:
I think it’s generally a good idea to leave a comment when marking a test exclusive that explains the reasoning. In this case: hardcoded portnumbers.

Reply:
Ok!